mods/persistence/modules/reagents/Chemistry-Holder.dm

```diff
@@ -5,4 +5,12 @@
 
 /datum/reagents/after_deserialize()
 	. = ..()
-	update_total()
\ No newline at end of file
+	update_total()
+
+//Reimplement this to get rid of the spam on save load
+/atom/create_reagents(var/max_vol)
```

Comment:
I assume this can't check for persistent_id since it fires most often in New()?

Reply:
I could probably check for it. Most times its used is during initialize.
I just don't particularly using persistent_id outside saving/init functions since there's no guarantees its gonna be set at the time the proc is called anyways.